python/src/WeibullMin.i

```diff
@@ -12,6 +12,6 @@ namespace OT { %extend WeibullMin { WeibullMin(const WeibullMin & other) { retur
 %pythoncode %{
 def Weibull(alpha=1.0, beta=1.0, gamma=0.0):
```

Comment:
I think it is better using `Weibull(*args*)`and check the argument

Reply:
ok, way simpler